server/odc-service/src/main/java/com/oceanbase/odc/service/permissionapply/project/ApplyProjectParameter.java

```diff
@@ -69,10 +68,6 @@ public static class ApplyResourceRole implements Serializable {
          * Resource role id, refer to {@link ResourceRole#getId()}
          */
         private Long id;
-        /**
-         * Resource role name, refer to {@link ResourceRole#getRoleName()}
-         */
-        private ResourceRoleName name;
```

Comment:
Please do not delete this field. The front-end need it to display.
BTW: here exists a bug, i will fix it soon...